R/rvar-print.R

```diff
@@ -65,11 +72,25 @@ format.rvar <- function(x, ..., summary = NULL, digits = 2, color = FALSE) {
 
 #' @rdname print.rvar
 #' @export
-str.rvar <- function(object, ..., summary = NULL, vec.len = NULL) {
+str.rvar <- function(
+  object, ..., summary = NULL, vec.len = NULL,
+  indent.str = paste(rep.int(" ", max(0, nest.lev + 1)), collapse = ".."),
+  nest.lev = 0, give.attr = TRUE
+) {
+
+  str_next <- function(x, ...) str(x, ...,
```

Comment:
the indentation is a little bit difficult to read. Is it worth adding formal `{}` to the function definition?

Reply:
sure, will do